<ui version="4.0" >
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWin</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QGroupBox" name="gb_send" >
      <property name="title" >
       <string>Send</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="lb_remoteAddress" >
         <property name="text" >
          <string>IP address:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="le_remoteAddress" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="lb_remoteAudioPort" >
         <property name="text" >
          <string>Audio base port:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="le_remoteAudioPort" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="lb_remoteVideoPort" >
         <property name="text" >
          <string>Video base port:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="le_remoteVideoPort" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="lb_sendConfig" >
         <property name="text" >
          <string>Codec config:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="le_sendConfig" />
       </item>
       <item row="4" column="0" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>261</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="pb_startSend" >
           <property name="text" >
            <string>&amp;Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_transmit" >
           <property name="text" >
            <string>&amp;Transmit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_stopSend" >
           <property name="text" >
            <string>St&amp;op</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>21</width>
             <height>28</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QGroupBox" name="groupBox_4" >
      <property name="title" >
       <string>Local video</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="PsiMedia::VideoWidget" native="1" name="vw_self" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" >
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Mic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sl_mic" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>96</height>
         </size>
        </property>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" >
     <widget class="QGroupBox" name="gb_receive" >
      <property name="title" >
       <string>Receive</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="lb_localAudioPort" >
         <property name="text" >
          <string>Audio base port:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="le_localAudioPort" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="lb_localVideoPort" >
         <property name="text" >
          <string>Video base port:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="le_localVideoPort" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="lb_receiveConfig" >
         <property name="text" >
          <string>Codec config:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="le_receiveConfig" />
       </item>
       <item row="3" column="0" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>261</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="pb_startReceive" >
           <property name="text" >
            <string>St&amp;art</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_stopReceive" >
           <property name="text" >
            <string>Sto&amp;p</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="title" >
       <string>Remote video</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="PsiMedia::VideoWidget" native="1" name="vw_remote" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2" >
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Spk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sl_spk" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>96</height>
         </size>
        </property>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>464</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About" />
   </widget>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_Configure" />
    <addaction name="separator" />
    <addaction name="action_Quit" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_About" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Quit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Configure" >
   <property name="text" >
    <string>Co&amp;nfigure Audio/Video...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PsiMedia::VideoWidget</class>
   <extends>QWidget</extends>
   <header>psimedia.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_remoteAddress</tabstop>
  <tabstop>le_remoteAudioPort</tabstop>
  <tabstop>le_remoteVideoPort</tabstop>
  <tabstop>le_sendConfig</tabstop>
  <tabstop>pb_startSend</tabstop>
  <tabstop>pb_transmit</tabstop>
  <tabstop>pb_stopSend</tabstop>
  <tabstop>le_localAudioPort</tabstop>
  <tabstop>le_localVideoPort</tabstop>
  <tabstop>le_receiveConfig</tabstop>
  <tabstop>pb_startReceive</tabstop>
  <tabstop>pb_stopReceive</tabstop>
  <tabstop>sl_mic</tabstop>
  <tabstop>sl_spk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
